src/main/java/seedu/address/model/reminder/ReminderOnOff.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.reminder;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the setting of a reminder, either on or off
+ */
+public class ReminderOnOff {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Setting of reminder should only contain true or false, "
+                    + "true to turn on reminder and false to turn off reminder.";
+
+    public final boolean setReminderBoolean;
+
+    /**
+     * Constructs a {@code ReminderOnOff}
+     *
+     * @param setReminder A valid value
+     */
+    public ReminderOnOff(String setReminder) {
+        requireNonNull(setReminder);
+        checkArgument(isValidSetReminder(setReminder), MESSAGE_CONSTRAINTS);
+        setReminderBoolean = setReminder.equals("true");
```

Comment:
I am wondering if this makes the setReminder field case sensitive (i.e if they input setReminder as True, it will modify setReminderBoolean to False)

Reply:
isValidSetReminder checks for the case sensitivity